presto-bytecode/src/main/java/com/facebook/presto/bytecode/ClassDefinition.java

```diff
@@ -37,6 +39,8 @@
 @NotThreadSafe
 public class ClassDefinition
 {
+    private static final int MAX_ALLOWED_VARARG_ARGUMENTS = 250;
```

Comment:
Where does this number come from? I didn't check myself, but https://stackoverflow.com/a/30581726/65458 mentions "256", "255" (and some parameter types count as 2, so "128", "127" could be also considered _mentioned").

Number aside, the constant & the check should be in the `MethodDefinition` class